base/src/com/google/idea/blaze/base/sync/projectstructure/ContentEntryEditor.java

```diff
@@ -68,6 +70,18 @@ public static void createContentEntries(
         contentEntry.addExcludeFolder(UrlUtil.fileToIdeaUrl(excludeFolder));
       }
 
+      File directory = new File(workspaceRoot.toString());
```

Comment:
I looked more into this and I think it is not needed to add this part inside the loop over `rootDirectories` because IIUC it only needs the `workspaceRoot`. I think it will be better if we removed this from here and instead modified [`ImportRoots$excludeProjectDataSubDirectory()`](https://github.com/bazelbuild/intellij/blob/master/base/src/com/google/idea/blaze/base/sync/projectview/ImportRoots.java#L141:L143) to be 
```
    private void excludeProjectDataSubDirectories() {
      excludeDirectoriesBuilder.addAll(
          BlazeDataStorage.ALL_PROJECT_SUBDIRECTORIES.stream()
              .map(d -> new WorkspacePath(d))
              .collect(toImmutableList()));
    }
```

I think this should be enough to add all the possible project subdirectories to the excluded list. If you think the current solution will still be needed, please let me know. Also this may require modifying some of the tests.